### PR TITLE
Add missing SNES controller profile

### PR DESCRIPTION
**Pull request by garbear**
This adds the SNES controller profile as a mandatory add-on. Let me be clear why:

The SNES controller is the archetypical retro controller. (Archetypical means "idealized model" or "most representative".) Indeed, it is the basis for the entire libretro controller abstraction.

That is why button combos in the GUI are shown using SNES buttons. In the future, we will show the button based on the controller in the user's hands. For first release, I hard-coded SNES buttons to keep things simple.

The first time the user is shown a button combo is the help dialog added in https://github.com/xbmc/xbmc/pull/14425. I made sure to include a large image so the user can visualize the button on their physical controller.

However, large images aren't helpful if they don't exist. Currently the user sees this empty dialog. This PR fixes that.

screen shot 2018-09-26 at 3 09 21 pm

## Motivation and Context
Forgot to include in https://github.com/xbmc/xbmc/pull/14425

## How Has This Been Tested?
Tested as part of https://github.com/xbmc/xbmc/pull/14425

## Screenshots (if appropriate):
After this PR:

help dialog

### CODE REVIEW

File: addons/game.controller.snes/addon.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="game.controller.snes"
+        name="SNES Controller"
+        version="1.0.6"
+        provider-name="Team Kodi">
+    <requires>
+    </requires>
```
Comment (Rechi) [inline addons/game.controller.snes/addon.xml]: as it is empty no need to list it

Comment (garbear) [reply addons/game.controller.snes/addon.xml]: This empty tag appears in all controllers in https://github.com/xbmc/repo-resources/tree/leia. Do you have a script that removes this?

Comment (Rechi) [reply addons/game.controller.snes/addon.xml]: Yes, will create a PR.

Comment (garbear) [reply addons/game.controller.snes/addon.xml]: SNES is now in repo-resources and master. So is 360. Should we remove SNES from repo-resources, or should we add 360 to repo-resources?

Comment (garbear) [reply addons/game.controller.snes/addon.xml]: NVM saw game.controller.default is already in repo-resources

**Comment by garbear** [comment]
@Rechi updated after https://github.com/xbmc/repo-resources/pull/57

**Pull request merged**